pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -182,6 +188,18 @@ type VaultAppRole struct {
 	SecretRef SecretKeySelector `json:"secretRef"`
 }
 
+type KubernetesAuth struct {
+	// The Kubernetes auth backend mount point if different from the default "kubernetes"
+	// +optional
+	Path string `json:"mountPath"`
```

Comment:
Is this pointing to the actual mount point for the kubernetes backend, or a particular path within it?
Also can you rephrase `if different from the default "kubernetes"` to say `if not specified, a default value of "..." will be used`